mlflow/ml-package-versions.yml

```diff
@@ -173,6 +173,24 @@ lightgbm:
     run: |
       pytest tests/lightgbm/test_lightgbm_autolog.py --large
 
+catboost:
+  package_info:
+    pip_release: "catboost"
+    install_dev: |
+      temp_dir=$(mktemp -d)
+      git clone --depth 1 https://github.com/catboost/catboost.git $temp_dir
+      cd $temp_dir/catboost/python-package
```

Comment:
nit: use pushd / popd

Reply:
Could you explain why?

Reply:
We can use a simple command:
`cd /tmp && pip install git+https://github.com/catboost/catboost.git#subdirectory=catboost/python-package`

Reply:
I think we need the entire directory to build the package

Reply:
https://catboost.ai/docs/installation/python-installation-method-build-from-source-linux-macos.html says we need to run the following command in 

```
../../../ya make -r -DUSE_ARCADIA_PYTHON=no -DUSE_SYSTEM_PYTHON=<Python version> [optional parameters]
```

Reply:
Don't need, it is included in setup.py script.

Reply:
Thanks for testing! I'll fix the code

Reply:
ah `git+https://github.com/catboost/catboost.git#subdirectory=catboost/python-package` means `git git+https://github.com/catboost/catboost.git` and then `python setup.py install` in `catboost/python-package`